### PR TITLE
Fixing can't create contract issue

### PR DESCRIPTION
**Pull request by morteza-binary**

### CODE REVIEW

File: www/templates/components/trades/payout.template.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <div class="col col-33 basis" ng-show="basis == 'payout'">{{'trade.payout' | translate}}</div>
     <div class="col col-33 basis" ng-show="basis == 'stake'">{{'trade.stake' | translate}}</div>
     <div class="col col-33 amount">
-        <input type="number" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
+        <input type="text" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
```
Comment (ashkanx) [inline www/templates/components/trades/payout.template.html]: If you change this from number to text, then will system display number keyboard or full keyboard?

Comment (morteza-binary) [inline www/templates/components/trades/payout.template.html]: Unfortunately summons full QWERTY Keyboard.

Comment (ashkanx) [inline www/templates/components/trades/payout.template.html]: Problem is when user enters "1." , we sent "amount":0, so server return error.
When user enters "1.", we should send  "amount":1, then it works as expected.

Comment (morteza-binary) [inline www/templates/components/trades/payout.template.html]: I have known the problem but unfortunately when we set input type to number
HTML engine doesn't accept anything else numbers and digits so when you
enter '1.' the engine ignore this value and we don't have access to it in
angular. So I cant find that what is the exact value in the input box.

**Pull request merged**